cms/templates/js/course_info_update.underscore

```diff
@@ -11,8 +11,14 @@
 		</div>
 		<div class="row">
 			<!-- cid rather than id b/c new ones have cid's not id's -->
-			<a href="#" class="save-button" name="<%= updateModel.cid %>">Save</a>
+			<a href="#" class="save-button" name="<%= updateModel.cid %>">Post</a>
 			<a href="#" class="cancel-button" name="<%= updateModel.cid %>">Cancel</a>
+            <%if (push_notification_enabled) { %>
+                <div class="new-update-push-notification">
+                    <label class="inline-label">Send push notification:</label>
+                    <input type="checkbox" class="toggle-checkbox" data-tooltip="<%= gettext('Send push notification to mobile devices') %>" checked />
```

Comment:
@lamagnifica Please review.
* Label for the checkbox:
  ** Send push notification:
* Tooltip for the checkbox:
  ** Send push notification to mobile devices